README.md

```diff
@@ -1,4 +1,50 @@
-# administrivia
-repository to track meta-work.
+# Administrivia
+
+This repository tracks PING's meta-work.
 
 2019 charter draft: https://w3cping.github.io/administrivia/charter-draft.html
+
+# PING Process
+
+PING's goal is to ensure that the specifications recommended by the W3C respect the privacy of people who use the web platform, and ensure that the web platform is an environment which allows people to browse the web privately.
+
+PING's core administrative mechanism for achieving this is _horizontal review_: members of PING review specifications and provide recommendations to the authors about how to ensure that their specifications support a web platform which respects privacy. This formal review process is the most visible part of a larger iceberg of activities aimed at helping specification authors write with privacy in mind. The best place to make privacy-related improvements to specifications is substantially in advance of horizontal review points. PING welcomes conversations with authors at all phases of specification development. Ideally, PING would be thoroughly familiar with specifications, their potential privacy impacts, and the measures takes to mitigate those impacts well before the time horizontal review points are reached, and those privacy reviews would consist mostly of codifying & recording conversations which have already happened, rather than pointing new things out to authors.
+
+PING's work extends substantially beyond the specific task of ensuring that a high privacy standard is upheld in those specifications which become recommendations. We also work with other privacy focused groups and researchers to gain better understanding of privacy risks and solutions for the web, ensure W3C processes protect privacy as a first-class feature of the Web. The remainder of this document focuses exclusively on the procedure of privacy reviews.
+
+## Privacy Review
+
+Every specification seeking recommendation status must be reviewed by PING. Specification authors request review by opening an issue in the [`w3cping/privacy-reviews`](https://github.com/w3cping) GitHub repository, using the supplied privacy review issue template. This issue should include as much context as possible so that PING can understand the aims and working environment of the specification.
```

Comment:
> Every specification seeking recommendation status must be reviewed by PING.

This sounds like something that needs be part of the [W3C's process document](https://www.w3.org/2019/Process-20190301/) in order to have any effect

Reply:
Absolutely.  A PING-published doc is the wrong place for this, except as it reflects changes otherwise made in the process.  Thanks for calling it out.